bricks/form-platform/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "@next-bricks/form-platform",
+  "version": "0.0.0",
+  "homepage": "https://github.com/easyops-cn/next-bricks/tree/master/bricks/form-platform",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:easyops-cn/next-bricks.git"
+  },
+  "license": "GPL-3.0",
+  "type": "module",
+  "sideEffects": true,
+  "files": [
+    "dist",
+    "dist-types",
+    "docs"
+  ],
+  "exports": {
+    "./package.json": "./package.json",
+    "./dist/bricks.json": "./dist/bricks.json",
+    "./dist/manifest.json": "./dist/manifest.json",
+    "./*": {
+      "types": "./dist-types/*/index.d.ts"
+    }
+  },
+  "scripts": {
+    "start": "cross-env NODE_ENV=development build-next-bricks --watch",
+    "build": "npm run build:main && npm run build:types",
+    "build:main": "cross-env NODE_ENV=production build-next-bricks",
+    "build:types": "tsc --emitDeclarationOnly --declaration --declarationDir dist-types --project tsconfig.json",
+    "build:manifest": "cross-env NODE_ENV=production build-next-bricks --manifest-only",
+    "test": "cross-env NODE_ENV='test' test-next",
+    "test:ci": "cross-env NODE_ENV='test' CI=true test-next",
+    "prepublishOnly": "cp package.json package.json.bak && npm pkg delete dependencies",
+    "postpublish": "mv package.json.bak package.json"
+  },
+  "dependencies": {
+    "@next-bricks/form": "^0.16.2",
+    "@next-core/element": "^1.1.2",
+    "@next-core/i18n": "^1.0.24",
+    "@next-core/react-element": "^1.0.10",
+    "@next-core/theme": "^1.1.1",
+    "react": "0.0.0-experimental-ee8509801-20230117"
+  },
+  "devDependencies": {
+    "@next-core/build-next-bricks": "^1.13.4",
```

```suggestion
    "@next-core/build-next-bricks": "^1.13.6",
```

修改后重新 yarn，应该不会有 yarn.lock 的更新